notebooks/official/model-registry/bqml-vertexai-model-registry.ipynb

```diff
@@ -0,0 +1,922 @@
+{
```

Comment:
<p>Global comment - we recommend you use "can" or "might" instead of "may", since this can get lost in translation and isn't as specific. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/842/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
Updated!